src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String HEADER = "fruit,quantity\n";

```

Comment:
```suggestion
    private static final String HEADER = "fruit,quantity";
```
Use System.lineSeparator()